tests/core/test_create_cmdline.py

```diff
@@ -93,6 +93,9 @@ def _mock_read_manifest(chip, manifest, clobber=False, clear=False):
     chip = siliconcompiler.Chip('test')
     for keypath in chip.allkeys():
         examples = chip.get(*keypath, field='example')
+        # TODO: undo this change once we support specifying pernode parameters using CLI
```

Comment:
What about optional?

Reply:
I don't think optional parameters would break this test since the CLI examples can just demonstrate how to set them globally (I'm envisioning that would look the same as it currently does)